pkg/cli/login/login.go

```diff
@@ -92,6 +104,11 @@ func NewCmdLogin(f kcmdutil.Factory, streams genericiooptions.IOStreams) *cobra.
 	cmds.Flags().BoolVarP(&o.WebLogin, "web", "w", o.WebLogin, "Login with web browser. Starts a local HTTP callback server to perform the OAuth2 Authorization Code Grant flow. Use with caution on multi-user systems, as the server's port will be open to all users.")
 	cmds.Flags().Int32VarP(&o.CallbackPort, "callback-port", "c", o.CallbackPort, "Port for the callback server when using --web. Defaults to a random open port")
 
+	if kcmdutil.FeatureGate("OC_ENABLE_WEB_LOGIN_NO_BROWSER").IsEnabled() {
+		cmds.Flags().BoolVar(&o.NoBrowser, "no-browser", o.NoBrowser, "Print the authorization URL instead of opening it in a browser. Only valid with --web.")
```

Comment:
I think we need some info about why do we need it. 

Because If we talk about standard login in OCP. I'm not sure this can be achieved. If yes, we can either re-use the one that was introduced for OIDC login https://github.com/openshift/oc/blob/a25e854ffddffcf2f8c1bc99b10afd1b34b8e0ca/pkg/cli/login/login.go#L101 or introduce an environment variable to disable this. cc'ing @liouk.

If we talk about OIDC login, it exists https://github.com/openshift/oc/blob/a25e854ffddffcf2f8c1bc99b10afd1b34b8e0ca/pkg/cli/login/login.go#L101 already.

Reply:
I'm using it mostly for Konflux